dgl_ptm/dgl_ptm/network/global_attachment.py

```diff
@@ -22,4 +22,5 @@ def global_attachment(agent_graph, ratio: float):
     agent_graph = AddReverse()(agent_graph)
 
     # Remove duplicate edges
-    agent_graph = dgl.to_simple(agent_graph, return_counts='cnt')
\ No newline at end of file
+    agent_graph = dgl.to_simple(agent_graph.to('cpu'), return_counts='cnt')
+    agent_graph = agent_graph.to(device)
```

Comment:
Can you please add an inline comment above these two lines and explain them?

Reply:
added comments